.changeset/deep-mangos-stand.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/datastore": minor
+"@fluid-private/test-end-to-end-tests": minor
+---
+---
+"section": feature
+---
+
+Shorter ids for Datastores and DDSes
+
+Fixed a long pending bug, which now enables us to use shorter ids for Datastores and DDSes when `enableRuntimeIdCompressor:"on"` is set in `IContainerRuntimeOptions`
```

Comment:
```suggestion
The Fluid Framework will now use shorter IDs for Datastores and DDSes when `enableRuntimeIdCompressor:"on"` is set in `IContainerRuntimeOptions`.
```

What's the benefit for customers? Space savings? Perf? Something else?

Reply:
Added more details about expected perf improvement.